scipy/spatial/qhull.pyx

```diff
@@ -2288,6 +2295,9 @@ class ConvexHull(_QhullUser):
         the nearest facets and nearest vertex indices.  Coplanar
         points are input points which were *not* included in the
         triangulation due to numerical precision issues.
+    good : Boolean array indicated which facets (simplices) are good.
+        Used with options that compute good facets, e.g. QGn and 
+        QG-n.
```

Comment:
As for the not-visible from inside the hull, the qhull documentation doesn't seem to describe this, but my interpretation is that the OUTER edge has to be visible and the inner edge doesn't count.  It also jives exactly with what I want to happen.  :->.  I was a really surprised by this at first but it makes sense if you take my interpretation.

Also, I have a lot of interesting examples to include in the documentation.  I will post a couple here.  3d examples are harder, but I have a couple where all the points are on the unit sphere that are kind of cool.  My programs are all dynamic in that I use the mouse or keyboard to cycle through facets and it will take a little work to make static documentation.  I only get to work on this here and there so it will take a couple of days but I will do it.

This really is a cool feature.  I am glad you are exercising it beyond what I thought of doing.

Reply:
I suspect we'd just want one simple / concise example for the docstring. If there are a lot of more interesting cases, I suppose those might be placed in a tutorial. The docs may need to clarify the inner / outer face distinction too, as this may cause confusion otherwise.

I provided a likely suitable file for placing the unit tests above.

Reply:
1.  I see the unit test reference now.  Sorry for the noise.
2. You are right about simple docstrings of course.
3. My comment about incorrect docstrings (issue 3 in the pull request) was incorrect and you are correct.  Just didn't read it right.

I don't know how much time I will have before the weekend but I can rebase, make some of your suggested changes, etc. as soon as I can.